pkg/karmadactl/addons/addons.go

```diff
@@ -0,0 +1,46 @@
+package addons
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"k8s.io/klog/v2"
+	"k8s.io/kubectl/pkg/util/templates"
+
+	"github.com/karmada-io/karmada/pkg/karmadactl/addons/install"
+)
+
+var (
+	addonsExamples = templates.Examples(`
+	# Enable or disable Karmada addons to the kubernetes cluster
```

Comment:
```suggestion
	# Enable or disable Karmada addons to the karmada-host cluster
```

I think this could be clearer. How do you think?

Reply:
Agree with you.
 I will modify it after your first round review.